src/appleseed/foundation/math/microfacet2.h

```diff
@@ -6,7 +6,7 @@
 // This software is released under the MIT license.
 //
 // Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
-// Copyright (c) 2014-2015 Francois Beaune, The appleseedhq Organization
```

Comment:
I take this opportunity to clarify **our** rules regarding copyright notices.

The general rule is as follow: **every source file and textual data file that we publish must have a copyright notice that extends from the initial commit in GitHub (2010) to now**. The copyright notice should have this exact form:

    // Copyright (c) 2010-YYYY <Author>, The appleseedhq Organization

There are two copyright holders here, `<Author>` and `The appleseedhq Organization`. The comma between `<Author>` and `The appleseedhq Organization` is a "and", not a "from".

Now, there is a special case to this rule for pre-2014 files: the organization switched from "Jupiter Jazz Limited" to "The appleseedhq Organization" in January 2014. Pre-2014 files must retain the original copyright notice and add the new one, so those files end up with two successive notices:

    // Copyright (c) 2010-2014 <Author>, Jupiter Jazz Limited
    // Copyright (c) 2014-YYYY <Author>, The appleseedhq Organization

The last point concerns `<Author>`: it is the full name of the initial author of the file. If the entire content of the entire file is replaced (which can also happen through successive move/rename file operations), it is possible to replace `<Author>` by the name of the actual author of the new content. There is no hard rules here, fairplay is *de rigueur*.

This explanation is now in the [wiki]().